sdcm/cluster.py

```diff
@@ -1566,15 +1566,15 @@ def update_db_binary(self, node_list=None):
             node_list = self.nodes
 
         new_scylla_bin = self.params.get('update_db_binary')
-        if new_scylla_bin:
+        if new_scylla_bin and os.listdir(new_scylla_bin):
```

Comment:
`new_scylla_bin` is a path to binary file, not directory. Why not use `os.path.exist(new_scylla_bin)`?

`os.listdir(new_scylla_bin)` might raise exception:

```
>>> os.listdir(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: coercing to Unicode: need string or buffer, NoneType found
>>> os.listdir('')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OSError: [Errno 2] No such file or directory: ''
```